org.hl7.fhir.utilities/src/test/java/org/hl7/fhir/utilities/graphql/GraphQLParserTests.java

```diff
@@ -0,0 +1,59 @@
+package org.hl7.fhir.utilities.graphql;
+
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class GraphQLParserTests {
+  private static final Logger ourLog = LoggerFactory.getLogger(GraphQLParserTests.class);
+  public static final String MULTIPLE_ID_VALUES_WITH_ID_AND_FAMILYGIVENNAMES =
+    "{Patient (id:[1,2,3]){id,name{family,given}}}";
+
+  @Test
+  public void testParseMultipleIdValuesAndSelectSetWithIdAndFamilyGivenNames() throws IOException, EGraphQLException, EGraphEngine {
```

Comment:
Please use the @DisplayName annotation from jupiter, so the name of the method isn't as huge